core/src/test/scala/org/apache/spark/status/AppStatusListenerSuite.scala

```diff
@@ -1275,6 +1275,49 @@ class AppStatusListenerSuite extends SparkFunSuite with BeforeAndAfter {
     assert(allJobs.head.numFailedStages == 1)
   }
 
+  test("SPARK-25451: total tasks in the executor summary should match total stage tasks") {
+    val testConf = conf.clone()
+      .set("spark.ui.liveUpdate.period", s"${Int.MaxValue}s")
```

Comment:
Use the config constant, like the existing code.

Reply:
Done.